src/style.css

```diff
@@ -1 +1,100 @@
 /* add styles here */
+@font-face {
+  font-family: Avenir;
+  src: url('fonts/Avenir-Book.ttf') format('truetype');
+  font-weight: normal;
+  font-style: normal;
+}
+
+@font-face {
+  font-family: Avenir;
+  src: url('fonts/Avenir-Heavy.ttf') format('truetype');
+  font-weight: bold;
+  font-style: normal;
+}
+
+body {
+  font-family: Avenir, Arial, Helvetica, sans-serif;
+}
+
+.search-bar__input {
+  display: block;
+  width: 100%;
+  margin-top: 20px;
+  font-family: Avenir, Arial, Helvetica, sans-serif;
+  border: 1px solid rgba(225, 231, 237, 1);
+  border-radius: 4px;
+}
+
+.search-bar__input--big {
+  height: 70px;
```

Comment:
You've fixed the height of the `.search-bar__input--big` class. Unless there's a specific requirement for this, it's better to let the content size dictate the height of the container. This helps to avoid overflow or accidental scroll bars.